### ISSUE TITLE
[BUG] [Linter] T027 False positive with golang comment syntax

### ISSUE THREAD

**Issue by silverwind** [Riverside-Healthcare/djLint#712]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: macOS 15
 - Python Version: 3.11
 - djLint Version: 1.31.1
 - template language: golang

## Issue
T027 likely shouldn't parse inside go template comments as it's picking up a `'` as a unbalanced string while this is purely comment text.

## How To Reproduce
```handlebars
{{/* can't */}}
```

## Contents of .djlintrc/pyproject.toml [tool.djlint]
```toml
[tool.djlint]
profile="golang"
```

**Comment by silverwind** [comment]
Thanks for your work as well!
